index.bs

```diff
@@ -2005,26 +2005,35 @@ New Charters and Major Changes</h4>
 	the [=Team=] <em class=rfc2119>must</em> extend the review period
 	to at least 60 days.
 
-	The Call for Review of a substantively modified charter
+	The Call for Review of a modified charter
 	<em class=rfc2119>must</em> highlight important changes
-	(e.g., regarding deliverables or resource allocation)
-	and include rationale for the changes.
+	(e.g., regarding changes in scope, deliverables, or resource allocation)
+	and <em class=rfc2119>must</em> include rationale for those changes.
 
-	The Call for Review of a new or substantively modified charter
-	<em class="rfc2119">must</em> include a disposition of comments for those received during the [=charter refinement=] process,
+	The Call for Review of a new or modified charter
+	<em class="rfc2119">must</em> include a disposition of comments received during the [=charter refinement=] process,
 	highlighting any issues that were closed despite sustained objections.
 
 <h4 id="charter-minor-changes">
 Minor Changes to Active Charters</h4>
 
-	Substantive changes to a [=charter=] (including [[#charter-extension|extensions]])
-	that do not affect the way the group functions in any significant way
+	[=Editorial changes=] or [=substantive changes=] to a [=charter=] (including [[#charter-extension|extensions]])
+	that do not affect the scope of the group's work
+	or the way the group functions in any significant way
+	are deemed <dfn>minor changes</dfn> and
 	<em class=rfc2119>may</em> be approved by a [=Team Decision=],
 	in which case they do not require [=charter refinement=] nor [=Advisory Committee Review=].
-	The following are examples of such changes:
-	the addition of an in-scope deliverable,
+	Any expansion of the scope of the [=charter=]
```

Comment:
```suggestion
	Any expansion or reduction of the scope of the [=charter=]
```

It could be that AC approval of a Charter is contingent on some item being in scope, and that if the group feels it cannot or will not proceed with that item, AC could be interested.

For example, consider some hypothetical charter that included 2 items in scope, one being some format for transferring sensitive information and another being a specification for how implementations can meet the privacy and security requirements for making such transfers. If the group decided to descope the latter, AC might well want them to stop working on the former too.

Reply:
I might suggest "any change to the scope of the charter" to make it more compact.

Reply:
```suggestion
	Any change to the scope of the [=charter=]
```